debian/apparmor/ubuntu_pro_apt_news.jinja2

```diff
@@ -20,6 +20,8 @@ profile ubuntu_pro_apt_news flags=(attach_disconnected) {
   # GH: 3079
   capability dac_override,
   capability kill,
+  capability chown,
+  capability fowner,
```

Comment:
Both of these were needed for `chown`?

Reply:
Yes, looks like it, but could use a second opinion in case this goes against recommended practice.

When testing with just `capability chown`, the tests on `apt_messages.feature` failed with another denial related to `fowner`. Adding both works

Reply:
Ok, thanks, that's good enough